main.go

```diff
@@ -282,3 +284,94 @@ func isTag(b *plugin.Build) bool {
 	return b.Event == plugin.EventTag ||
 		strings.HasPrefix(b.Ref, "refs/tags/")
 }
+
+// execute executes all the commands given using specified directory as CWD
+func execute(cmds []*exec.Cmd, cwd string) error {
+	for _, cmd := range cmds {
+		cmd.Dir = cwd
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+		trace(cmd)
+		err := cmd.Run()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// Describes .gitattribute tweaking rules in form
+// RE pattern => replacement string (literal)
+type tweak struct {
+	pattern     *regexp.Regexp
+	replacement string
+}
+
+// .gitattribute tweaking rules
+var tweaks = make([]tweak, 0, 3)
```

Comment:
If this will only ever have 3 elements, you could also use an array instead of a slice.